MeshTweaker.py

```diff
@@ -3,7 +3,7 @@
 import math
 from time import time, sleep
 import re
-import os
+# import os
```

Comment:
The os module is used later. Or is this a redundant import?

Reply:
Tweaker.py already imports os, and in MeshTweaker.py it's redundant.
Actually instead of commenting, let me remove the line. Also, while we're on the subject, I'll remove redundant imports from other files. I'm not really keen on removing code where I'm not one of the official developers but I guess final word is still up to you anyway :)

Reply:
Oh, by the way... Nice Utility. Totally forgot to say that to begin with. I think I'm going to build something based on it, or extend it :)